client/dashboard/sites/index.tsx

```diff
@@ -55,25 +55,31 @@ const fields = [
 		id: 'backups',
 		label: __( 'Backups' ),
 		getValue: ( { item }: { item: Site } ) =>
-			item.plan.features.active.includes( 'backups' ) ? 'enabled' : 'disabled',
+			item.plan.features.active.includes( 'backups' ) || undefined,
 		elements: [
-			{ value: 'enabled', label: __( 'Enabled' ) },
-			{ value: 'disabled', label: __( 'Disabled' ) },
+			{ value: true, label: __( 'Enabled' ) },
+			{ value: undefined, label: __( 'Disabled' ) },
```

Comment:
@ellatrix: curious: did it not work with `false`?

Reply:
I had asked about this in Slack. The problem is that if you first add a filter, its value is undefined by default until you set the value, and I can't find how to set a default value. So I opted here to make one of the values undefined, otherwise you have zero results when you add a filter at first.